WORKSPACE.bazel

```diff
@@ -34,3 +34,7 @@ grpc_deps()
 load("@com_github_grpc_grpc//bazel:grpc_extra_deps.bzl", "grpc_extra_deps")
 
 grpc_extra_deps()
+
+load("@io_opentelemetry_cpp//bazel:repository.bzl", "opentelemetry_cpp_deps")
```

Comment:
Will we need to change all the quickstart WORKSPACE files too?  Not now, but it a future PR?
Maybe it is time to create the multiple workspace functions like TensorFlow does.

Reply:
> Will we need to change all the quickstart WORKSPACE files too? Not now, but it a future PR?

Yeah, we will. That is a task for Milestone 2.

> Maybe it is time to create the multiple workspace functions like TensorFlow does.

:shrug: